runtime/connectors/https/https.go

```diff
@@ -70,6 +70,9 @@ func (c connector) ConsumeAsFiles(ctx context.Context, env *connectors.Env, sour
 		return nil, fmt.Errorf("failed to fetch url %s:  %w", conf.Path, err)
 	}
 	defer resp.Body.Close()
+	if resp.StatusCode >= http.StatusMultipleChoices {
```

Comment:
Can we make it `resp.StatusCode < 200 || resp.StatusCode >= 300`?

There are a few `100` that theoretically could be encountered. And I feel using numbers for range checks versus aliases is more meaningful (I had to double-check that `MultipleChoices` was the alias for 300).